hazelcast/src/main/java/com/hazelcast/spi/StatisticsService.java

```diff
@@ -0,0 +1,14 @@
+package com.hazelcast.spi;
+
+import com.hazelcast.monitor.LocalInstanceStats;
+
+import java.util.Map;
+
+/**
+ * An interface to give SPI services ability to publish their statistics.
+ */
+public interface StatisticsService {
```

Comment:
I don't think it is the correct name. It is an ability  and I would put it in line with e.g. MigrationAwareService. So I would call this interface 'StatisticsAwareService'.